contracts/axelar-operators/src/contract.rs

```diff
@@ -131,3 +131,30 @@ impl AxelarOperators {
         Ok(res)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use axelar_soroban_std::assert_some;
+    use soroban_sdk::testutils::Address as _;
+    use soroban_sdk::{Address, Env};
+
+    use super::{AxelarOperators, AxelarOperatorsClient, DataKey};
+
+    #[test]
+    fn test_initialize() {
+        let env = Env::default();
+        let contract_id = env.register_contract(None, AxelarOperators);
+        let client = AxelarOperatorsClient::new(&env, &contract_id);
+        let user = Address::generate(&env);
+
+        client.initialize(&user);
+
+        assert_some!(env.as_contract(&contract_id, || {
+            env.storage()
+                .instance()
+                .get::<DataKey, bool>(&DataKey::Initialized)
+        }));
```

Comment:
```suggestion
        assert_env!(contract_id, assert_some!(
            env.storage()
                .instance()
                .get::<DataKey, bool>(&DataKey::Initialized)
        )));
```
wonder if we should some macro like this for convenience. `env` can be assumed to exist in the scope already perhaps. Otherwise, move the `assert_some` to inside the function for consistency

Reply:
can be done in a separate PR though if we want to add this

Reply:
having it look exactly like that would require a proc macro since regular macros are hygenic and can't pull in variables from outside their context. will save it for a future PR since most instances of the `env.as_contract()` pattern occur in `auth.rs` which we're planning to rework (and probably removing most of its uses).

handled the second part in [f04854c](https://github.com/axelarnetwork/axelar-cgp-soroban/pull/49/commits/f04854c5df691ac200ccc91c72ca7b32900e28ea)